libraries/eosiolib/contract.hpp

```diff
@@ -26,7 +26,7 @@ class contract {
        * @brief Construct a new contract object.
        * @param n - The name of this contract
        */
-      contract( name n ):_self(n) {}
+      contract( name n, name code ):_self(n), _code(code) {}
```

Comment:
Add code to doxygen comment (see comment below regarding how to describe `code`). Also I would like to rename `n` to something more meaningful like `receiver`.